extract_from_hg.py

```diff
@@ -195,15 +196,110 @@ def commitCset(cset):
     runCommand(['git', 'commit', '-m', commitMsg, '--author=' + cset.user(),
                 '--date=' + str(csetDate)])
 
+def formatCommitMessageLine(line):
+    # Now do translations on the line.
+
```

Comment:
Most of the function is replacements rather than extraction. I think using an extraction technique would actually end up with more complexity than we have now.